buku

```diff
@@ -2114,9 +2138,6 @@ class BukuDb:
             return False
 
         if self.delete_rec_all():
-            with self.lock:
-                self.cur.execute('VACUUM')
-                self.conn.commit()
```

Comment:
`VACUUM` is now invoked directly by `delete_rec_all()` after the commit (same as for other deletion operations)